rclcpp/include/rclcpp/callback_group.hpp

```diff
@@ -28,18 +28,28 @@ namespace rclcpp
 {
 
 // Forward declarations for friend statement in class CallbackGroup
-namespace node {class Node;}
-namespace executor {class Executor;}
+namespace node
+{
+class Node;
+} // namespace node
+namespace executor
+{
+class Executor;
+} // namespace executor
 
 namespace callback_group
 {
 
-enum class CallbackGroupType {MutuallyExclusive, Reentrant};
+enum class CallbackGroupType {
+  MutuallyExclusive,
+  Reentrant
+};
 
 class CallbackGroup
 {
   friend class rclcpp::node::Node;
   friend class rclcpp::executor::Executor;
+
 public:
```

Comment:
What about the whitespace before these? I would personally prefer it to be horizontally aligned with the `class` keyword. I don't like having leading whitespace which is not a multiple of 2 and does not horizontally align with anything above or below it.

I would propose to have no extra space before the access modifiers.

However, that's just my opinion, what do you guys think?